lib/packwerk/generators/templates/package.yml

```diff
@@ -11,6 +11,10 @@ enforce_dependencies: true
 # We recommend enabling this for any new packages you create to aid with encapsulation.
 enforce_privacy: false
 
+# By default the public path will be app/public/, however this may not suit all applications' architecture so
+# this allows you to modify what your package's public path is.
+public_path: app/public/
```

Comment:
My gut feeling is that this should be commented out rather than set to the default. WDYT @thegedge?

Reply:
👍 Yeah, commented out would be good.

I think it makes sense to have it not commented out if there's no default in code, but (for the time being) we will want to keep the default in code (a lot of internal things will depend on that). It's also not useful unless `enforce_privacy: true` was the default 🙂

(sorry, I type a lot of words to say I agree...)

Reply:
Done!